torch/_dynamo/skipfiles.py

```diff
@@ -196,6 +196,11 @@ def _module_dir(m: types.ModuleType):
     _module_dir(torch) + "_export/wrappers.py",
 }
 
+if torch.distributed.is_available() and torch._dynamo.config.trace_distributed:
+    FILENAME_INLINELIST |= set(
+        glob.glob(_module_dir(torch) + "distributed/**/*.py", recursive=True),
+    )
+
```

Comment:
Can you add ```torch.distributed``` into the ```SUBMODULE_INLINELIST```? That would be a more easy way to force inline all files under a submodule.

Reply:
No, this fails for whatever reason.

Reply:
If you want a crack at debugging:

1) voz/fsdp_autograd3
2) build
3) torchrun --standalone --nproc_per_node=2 fsdp.py

Reply:
I think this would break trunk since you land after #110835. Do you mind to send a forward fix? Or I can help to forward fix it.